todo_app/app.py

```diff
@@ -1,15 +1,27 @@
-from flask import Flask
-
+from flask import Flask, render_template
+from werkzeug.utils import redirect
+from todo_app.data.session_items import *
 from todo_app.flask_config import Config
+from flask import request 
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
+
 @app.route('/')
 def index():
-    return 'Hello World!'
+    todoitems = get_items()
+    return render_template('index.html', todoitems = todoitems)
 
+@app.route('/add', methods=['POST'])
+def add():
+    if request.method == 'POST':
+        add_item(title=request.form.get('title'))
+        return redirect("http://127.0.0.1:5000/")
```

Comment:
Here, we're redirectly to a full URL, which means this wouldn't work outside of local development. If we actually just specify the path and not the domain, the redirect will work more widely, e.g.
`return redirect("/")`